utxo/src/storage/in_memory.rs

```diff
@@ -31,15 +30,15 @@ pub struct UtxosDBInMemoryImpl {
 }
 
 impl UtxosDBInMemoryImpl {
-    pub fn new(best_block: Id<GenBlock>, initial_utxos: BTreeMap<OutPoint, Utxo>) -> Self {
+    pub fn new(best_block: Id<GenBlock>, _initial_utxos: BTreeMap<OutPoint, Utxo>) -> Self {
         Self {
             store: BTreeMap::new(),
             undo_store: BTreeMap::new(),
             best_block_id: best_block,
         }
     }
 
-    pub(crate) fn internal_store(&mut self) -> &BTreeMap<OutPoint, Utxo> {
+    pub fn internal_store(&mut self) -> &BTreeMap<OutPoint, Utxo> {
```

Comment:
Why did you replace pub(crate) with pub? This shouldn't be part of the public interface.

Reply:
in_memory mod is not exposed to the public and is kept only for tests. So this seems like the right thing to do just to keep it clean.

Reply:
I get that, but in-memory should be exposable to the public if we feel the need for it. Having pub(crate) signals that this is a special method that shouldn't be used normally. Even better, if you can change this to pub(super) or something more restrictive, I'm OK with that..

Reply:
You know, even better, maybe just add `#[cfg(test)]` above it. Then you can keep it as pub if you wish.